centreon/www/front_src/src/ResourceAccessManagement/Listing/Listing.tsx

```diff
@@ -0,0 +1,42 @@
+import { MemoizedListing } from '@centreon/ui';
+
+import useListing from './useListing';
+
+const ResourceAccessRulesListing = (): JSX.Element => {
+  const listing = useListing();
+
```

Comment:
You can use object destructuring to directly destructure all the props that you need.
`const {loading , page, ....} = useListing()`

Reply:
Done